### DISCUSSION TITLE
Run armo portal locally

### DISCUSSION THREAD

**Discussion by ZacHaque** [kubescape/kubescape#326] [Ideas]
I am using kubescape for code scanning and also running as cronjob in the cluster, this is really good. Recently I have looked at image scanning and RBAC analysis look fantastic. However, did anyone though about running portal locally then armo portal on the internet. Some organization would not like to send the scanning data out to world.

**Comment by dwertent** [comment]
Hi @ZacHaque , I apologize for not answering earlier, I did not notice this question.
Thank you for your feedback, I hope you still use Kubescape and that you find it helpful. Please do not hesitate to reach out for any further questions or support.

Regarding your question, Yes! we have in our roadmap open-sourcing the portal and all other components. Unfortunately there is a lot of work to be done before we open source the portal so this might take some time.

Regarding your concern about sending data - It is a valid concern, but I can assure you and others, we are not sending any confidential data so as the secrets data, configMaps data and the values of the environments. 
Locally for me Kubescape is open-source so you can see this on yourself :)
